tests/queries/0_stateless/02340_parts_refcnt_mergetree.sh

```diff
@@ -58,7 +58,7 @@ function check_refcnt_for_table()
     $CLICKHOUSE_CLIENT -q "select table, name, refcount>=6 from system.parts where database = '$CLICKHOUSE_DATABASE' and table = '$table' and refcount >= 3"
 
     # Kill the query gracefully.
-    kill -INT $PID
```

Comment:
The test works wrong if the query does not exist, instead, it's execution time should be increased, just replace `numbers(200)` with `numbers(2000)` and update the comment about max execution time to 50 seconds

And this should be enough, this is from the original failure:

```
2024.06.03 08:50:35.761910 [ 9214 ] {data_02340-uyfoemvoer} <Debug> executeQuery: (from [::1]:52074) (comment: 02340_parts_refcnt_mergetree.sh) select sleepEachRow(0.1) from data_02340 where part = 1 (stage: Complete)
...
2024.06.03 08:50:40.918544 [ 9214 ] {data_02340-uyfoemvoer} <Debug> TCPHandler: Processed in 5.176384591 sec.
...
2024.06.03 08:50:41.692615 [ 9214 ] {a74f4b5f-50cb-45ef-a42d-6d1e9b05b17d} <Debug> executeQuery: (from [::1]:52102) (comment: 02340_parts_refcnt_mergetree.sh) select table, name, refcount>=6 from system.parts where database = 'test_eyer7bd5' and table = 'data_02340' and refcount >= 3 (stage: Complete)

```

Reply:
ok, thx, hope it is good now